provider/cldr/src/transform/time_zones/mod.rs

```diff
@@ -258,7 +261,7 @@ mod tests {
             .unwrap();
         assert_eq!(
             "PDT",
-            specific_names_short.get()["America_Pacific"]["daylight"]
+            specific_names_short.get()["America_Pacific"][&"daylight".parse::<TinyStr8>().unwrap()]
```

Comment:
`question`: can you use `tinystr8!` macro here?

Reply:
`follow-up`: Definitely! That's a great idea. I forgot there were macros for these. I'll make the change.